targets/XPlatCppSdk/source/code/include/playfab/QoS/QoS.h

```diff
@@ -23,5 +23,14 @@ namespace PlayFab
 
         // Number of times a server is pinged
         constexpr int NUM_OF_PING_ITERATIONS = 3;
+
+        enum QoSErrorCode
```

Comment:
>QoSErrorCode [](start = 13, length = 12)

Can we please add a comment for this enum explaining that these are possible error code values for QoSResult.erroCode?

Reply:
I will add commnents